### ISSUE TITLE
BUG: default avatar different for new chat and In LHN reported by @gadhiyamanan

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12022]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. click on :green-plus-button: icon
2. select new chat
3. search for random new user ( like [g1@gmail.com](mailto:g1@gmail.com))
4. check the avatar (avatar background color)
5. click on the user to initiate conversation
6. check the avatar (avatar background color)  on the conversation or LHN

## Expected Result:
default avatar should be same while searching for new chat and LHN

## Actual Result:
default avatars are different

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.18-2
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/196825164-55d43099-5b26-40a9-b9c4-e05e113621d0.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @gadhiyamanan 
**Slack conversation:** 
https://expensify.slack.com/archives/C01GTK53T8Q/p1666160339859479
[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

https://user-images.githubusercontent.com/43996225/196825188-60c818f5-3aa9-4bc3-8b60-7ff5c99463ac.mp4

**Comment by ntrepanier** [comment]
I tried to recreate the issue following the listed steps on android and web in both production and staging but could not reproduce.

**Comment by aldo-expensify** [comment]
Just noticed that there are some issues assigned to @ntrepanier , reapplying the `AutoAssignerTriage`!

**Comment by marcaaron** [comment]
I would recommend we not fix this. It's not really a "bug" and an intentional behavior that we've had for a long time. If there's no avatar to use a random avatar is generated. The fact that it is changing color doesn't feel like a real problem we need to solve and fixing it brings very little value.

**Comment by marcaaron** [comment]
@aldo-expensify I think this is related to the change here https://github.com/Expensify/App/pull/11400

The server is still sending a md5 hash 

https://github.com/Expensify/Web-Expensify/blob/03510d77cd89d1cd6311bd9189e4c4306d97612b/lib/User/Utils.php#L327-L333

But in the client we are doing something else so the avatar colors change now.

**Comment by aldo-expensify** [comment]
@marcaaron you are totally right, good finding!

**Comment by aldo-expensify** [comment]
I think this may be the equivalent function in php: https://stackoverflow.com/a/15702546/16434681

**Comment by sophiepintoraetz** [comment]
So @marcaaron - did we decide we're going to fix this or this is intended behaviour?

**Comment by marcaaron** [comment]
I am sideways thumb on whether this is worth prioritizing (we just have so many more important issues to work on right now). But, maybe a good first pick issue to do eventually? Doesn't feel like an urgent `Bug` so I think we can just get to it when we get to it.

**Comment by sophiepintoraetz** [comment]
Not a priority.

**Comment by grgia** [comment]
I might be able to fix this while I implement the new avatars for new dots, I think we just need to match the hashing function in Web with the one in App. Mind if I assign for now @marcaaron ?

**Comment by grgia** [comment]
Not overdue, on hold

**Comment by sophiepintoraetz** [comment]
What are we on hold for @grgia? Mind linking the GH its dependent on? That way we can close this issue and reopen it once the other is done

**Comment by grgia** [comment]
I was holding the avatar changes to complete [core branding changes](https://github.com/Expensify/App/issues/12251) first. However that's almost done, so I should be able to work on avatar fixes again

**Comment by grgia** [comment]
Still holding, planning to work on avatars next week!

**Comment by grgia** [comment]
Adding reviewing label

**Comment by MelvinBot** [comment]
This issue has not been updated in over ***15*** days. @grgia, @sophiepintoraetz eroding to ***Monthly*** issue.

P.S. Is everyone reading this sure this is really a near-term priority? Be brave: if you disagree, go ahead and close it out. If someone disagrees, they'll reopen it, and if they don't: one less thing to do!

**Comment by grgia** [comment]
closing as completed, since no longer occurs with the new default avatars on new dot!

**Comment by gadhiyamanan** [comment]
@grgia can you please reopen issue as reporting bonus is pending

**Comment by sophiepintoraetz** [comment]
@gadhiyamanan - I don't think you're eligible for a reporting bonus here, as we updated our brand on a wholesale level, which means the overall design of our chats have changed. I'm going to confirm this internally but wanted to let you know.

**Comment by gadhiyamanan** [comment]
> I might be able to fix this while I implement the new avatars for new dot, I think we just need to match the hashing function in Web with the one in App. Mind if I assign for now @marcaaron ? 

Please check https://github.com/Expensify/App/issues/12022#issuecomment-1332901377

**Comment by sophiepintoraetz** [comment]
See the following comment @gadhiyamanan 
>I was holding the avatar changes to complete https://github.com/Expensify/App/issues/12251 first. However that's almost done, so I should be able to work on avatar fixes again

**Comment by grgia** [comment]
You're correct @sophiepintoraetz  we didn't end up needing to fix this, the bug was made obsolete by the brand change.

**Comment by sophiepintoraetz** [comment]
Thanks for confirming, Georgia. @gadhiyamanan - apologies but there is no reporting bonus to be issued here. I see you have been prolific in other areas, no doubt more will crop up for you to report!
